docs/core/diagnostics/distributed-tracing-instrumentation-walkthroughs.md

```diff
@@ -455,3 +455,6 @@ void DoBatchWork(ActivityContext[] requestContexts)
 
 Unlike events and Tags that can be added on-demand, links must be added during StartActivity() and
 are immutable afterwards.
+
+> [!IMPORTANT]  
+> According to the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#span-limits), the number of links is capped at 128 by default.
```

Comment:
This creates the impression that the limit is actively enforced, which is not accurate. It would be beneficial to clarify this by stating, `As per the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#span-limits), the suggested limit for the number of links is 128, but it's important to note that this limit is not enforced.`

CC @noahfalk